configs/components/runtime.rb

```diff
@@ -23,6 +25,16 @@
   if platform.is_aix?
     pkg.install_file File.join(libdir, "libstdc++.a"), "/opt/puppetlabs/puppet/lib/libstdc++.a"
     pkg.install_file File.join(libdir, "libgcc_s.a"), "/opt/puppetlabs/puppet/lib/libgcc_s.a"
+  elsif platform.is_windows?
+    arch = platform.architecture == "x64" ? "64" : "32"
+    lib_type = platform.architecture == "x64" ? "seh" : "sjlj"
+    pkg.install_file "/cygdrive/c/tools/mingw#{arch}/bin/libgcc_s_#{lib_type}-1.dll", settings[:bindir]
+    pkg.install_file "/cygdrive/c/tools/mingw#{arch}/bin/libstdc++-6.dll", settings[:bindir]
+    pkg.install_file "/cygdrive/c/tools/mingw#{arch}/bin/libwinpthread-1.dll", settings[:bindir]
```

Comment:
We have a function for the first part of this path, right? And we can probably normalize the entire binpath here.